docs/3.api/5.kit/2.programmatic.md

```diff
@@ -53,24 +53,20 @@ Loading conditions for Nuxt. `loadNuxt` uses [`c12`](https://github.com/unjs/c12
   
   If set to `true`, Nuxt will be ready to use after the `loadNuxt` call. If set to `false`, you will need to call `nuxt.ready()` to make sure Nuxt is ready to use.
 
-- `rootDir` (optional)
+- `cwd` (optional)
```

Comment:
These two parameters are both documented in the link above (https://github.com/unjs/c12).

This is meant to be a list of _additional_ options that are respected.

Instead, shall we just remove them?